perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/DatabaseManagerMethod.java

```diff
@@ -50,5 +52,21 @@ public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 			return ac.getDatabaseManager().getDatabaseInformation(ac.getSession());
 		}
+	},
+
+	/**
+	 *	Get time in ns "nanoseconds" of calling 1 simple update query to DB.
+	 *  This query will update property for this purpose in configurations table.
+	 *
+	 * @exampleResponse "196920"
+	 * @return time of processing query in nanoseconds
+	 */
+	getTimeOfQueryPerformance {
+
+		@Override
+		public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
```

Comment:
Please change the return type of the method to Long:
```suggestion
		public Long call(ApiCaller ac, Deserializer parms) throws PerunException {
```

Reply:
Done.